storm-server/src/main/java/org/apache/storm/daemon/drpc/DRPC.java

```diff
@@ -208,6 +208,7 @@ public void failRequest(String id, DRPCExecutionException e) throws Authorizatio
         }
     }
 
+    //Should this be changed to private?
```

Comment:
Leave this to the comment on PR or file an issue. TODO-like comment is tend to not be addressed afterwards.